charts/kyverno/ci/webhooks-values.yaml

```diff
@@ -0,0 +1,9 @@
+config:
```

Comment:
Can probably remove this file since it was initially added to validate the more complex logic in NOTES.txt.

Reply:
Removed.